### PR TITLE
Fixed a variety of small issues

### PR DESCRIPTION
**Pull request by NelsonTheBrave**
Collection of small fixes:

- Css for Stora Salongen
- Fetching does not include "localhost" to work on external deploy services
- Clarified names of methods and files referring to latestScreenings and sepcificScreenings
- Added error handling on latestScreenings route and array cleaning

### CODE REVIEW

File: static/src/assets/Logo.PNG
Comment (JrSchoolAccount) [inline static/src/assets/Logo.PNG:1]: This can be deleted.
Its not used in the project :)

**Comment by FelixHultman** [comment]
looks good from what i can see, good work with fixing our problems with the names, looks much better now!

**Review by FelixHultman — APPROVED** [review]

File: static/src/assets/tandem-bike.png
Comment (JrSchoolAccount) [inline static/src/assets/tandem-bike.png:1]: Remove this :)

File: static/tandem-bike.png
Comment (JrSchoolAccount) [inline static/tandem-bike.png:1]: This can be removed :)

**Review by JrSchoolAccount — APPROVED** [review]
Keep up the good work :) 
Suggested som changes, other than that it looks good ^^
No need to request changes.

**Pull request merged**